### ISSUE TITLE
CWInitConvTemp instead of HWInitConvTemp for calculating hot water density

### ISSUE THREAD

**Issue by ecoeficiente** [NREL/EnergyPlus#6502]
Issue overview
--------------
CWInitConvTemp is used instead of HWInitConvTemp when calculating water rho value for hot water in HWBaseboardRadiator.cc, BaseboardRadiator.cc, Boilers.cc, etc.

It may be intentional in order to work with standard density. However, I think it is worth checking because I don't see much logic in calculating specific heat with hot temperature and density with cold temperature as in HWBaseboardRadiator.cc:
```
Cp = GetSpecificHeatGlycol( PlantLoop( HWBaseboard( BaseboardNum ).LoopNum ).FluidName, HWInitConvTemp, PlantLoop( HWBaseboard( BaseboardNum ).LoopNum ).FluidIndex, RoutineName );
rho = GetDensityGlycol( PlantLoop( HWBaseboard( BaseboardNum ).LoopNum ).FluidName, CWInitConvTemp, PlantLoop( HWBaseboard( BaseboardNum ).LoopNum ).FluidIndex, RoutineName );
WaterVolFlowRateMaxDes = DesCoilLoad / ( PlantSizData( PltSizHeatNum ).DeltaT * Cp * rho );
```

**Comment by Nigusse** [comment]
This issue is duplicate of #5821 . Any one working on this issue?

**Comment by RKStrand** [comment]
@Nigusse If you are already working on #5821 (you are shown as the assignee), then I think you can link this to that fix.  If you are looking for someone else to take this over, let me know and I will put it on my list of things to fix.

**Comment by Nigusse** [comment]
@RKStrand I am already working on #5821 and will link this to that. Thanks.

**Comment by RKStrand** [comment]
@Nigusse Sounds good--thanks!

**Comment by Myoldmopar** [comment]
Closed via #6813
